console.go

```diff
@@ -316,46 +322,86 @@ func consoleDefaultFormatTimestamp(timeFormat string, noColor bool) Formatter {
 	}
 }
 
-func consoleDefaultFormatLevel(noColor bool) Formatter {
+func consoleDefaultFormatLevel(noColor bool, levelAbbreviator Formatter) Formatter {
 	return func(i interface{}) string {
 		var l string
 		if ll, ok := i.(string); ok {
 			lvl, err := ParseLevel(ll)
 			if err != nil {
-				l = colorize("???", colorBold, noColor)
+				l = colorize(levelAbbreviator(ll), colorBold, noColor)
 			}
 
 			switch lvl {
-			case TraceLevel:
-				l = colorize("TRC", colorMagenta, noColor)
 			case DebugLevel:
-				l = colorize("DBG", colorYellow, noColor)
+				l = colorize(levelAbbreviator(lvl), colorYellow, noColor)
+			case TraceLevel:
+				l = colorize(levelAbbreviator(lvl), colorMagenta, noColor)
 			case InfoLevel:
-				l = colorize("INF", colorGreen, noColor)
+				l = colorize(levelAbbreviator(lvl), colorGreen, noColor)
 			case WarnLevel:
-				l = colorize("WRN", colorRed, noColor)
+				l = colorize(levelAbbreviator(lvl), colorRed, noColor)
 			case NotifyLevel:
-				l = colorize("NTF", colorRed, noColor)
+				l = colorize(levelAbbreviator(lvl), colorRed, noColor)
 			case ErrorLevel:
-				l = colorize(colorize("ERR", colorRed, noColor), colorBold, noColor)
+				l = colorize(colorize(levelAbbreviator(lvl), colorRed, noColor), colorBold, noColor)
 			case CriticalLevel:
-				l = colorize(colorize("CRT", colorRed, noColor), colorBold, noColor)
+				l = colorize(colorize(levelAbbreviator(lvl), colorRed, noColor), colorBold, noColor)
 			case AlertLevel:
-				l = colorize(colorize("ALR", colorRed, noColor), colorBold, noColor)
+				l = colorize(colorize(levelAbbreviator(lvl), colorRed, noColor), colorBold, noColor)
 			default:
-				l = colorize("???", colorBold, noColor)
+				l = colorize(levelAbbreviator(lvl), colorBold, noColor)
 			}
 		} else {
 			if i == nil {
-				l = colorize("???", colorBold, noColor)
+				l = colorize(levelAbbreviator(i), colorBold, noColor)
 			} else {
-				l = strings.ToUpper(fmt.Sprintf("%s", i))[0:3]
+				l = levelAbbreviator(i)
 			}
 		}
 		return l
 	}
 }
 
+func consoleDefaultFormatLevelAbbreviation() Formatter {
+	return func(i interface{}) string {
+
+		if i == nil {
+			return "???"
+		}
+
+		if _, ok := i.(string); ok {
+			// This case corresponds with a non-level string
+			return "???"
+		}
+
+		if level, ok := i.(Level); ok {
+			switch level {
+			case DebugLevel:
+				return "DBG"
+			case TraceLevel:
```

Comment:
I am ok with this change but this is to show [TRC] correct? Like this way in the old way?

22:24:13 [I] Reconnect in progress, skipping connection attempt.

Reply:
Maybe we want `[]` square brackets around them.

Reply:
Correct, this whole change was to make it possible for us to override the abbreviation used by the console writer. By default, zerolog would print `TRC` as the severity, so this method (`consoleDefaultFormatLevelAbbreviation`) was created to keep the default behaviour. On thoth's side, we have an implementation that changes the abbreviation to `[T]`